spec/lib/rbac/filterer_spec.rb

```diff
@@ -1197,100 +1197,123 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
       end
     end
 
-    context 'with cloud network and network manager' do
+    context "with cloud network and network manager" do
       let!(:network_manager)   { FactoryGirl.create(:ems_openstack).network_manager }
-      let!(:cloud_network)     { FactoryGirl.create(:cloud_network, :ext_management_system => network_manager) }
       let!(:network_manager_1) { FactoryGirl.create(:ems_openstack).network_manager }
-      let!(:cloud_network_1)   { FactoryGirl.create(:cloud_network, :ext_management_system => network_manager_1) }
 
-      context 'with belongs_to_filter' do
+      context "with belongs_to_filter" do
         before do
           group.entitlement = Entitlement.new
           group.entitlement.set_managed_filters([])
           group.entitlement.set_belongsto_filters(["/belongsto/ExtManagementSystem|#{network_manager.name}"])
           group.save!
         end
 
-        context 'when records match belognsto filter' do
-          it 'lists cloud networks with network manager according to belongsto filter' do
-            User.with_user(user) do
-              results = described_class.search(:class => CloudNetwork).first
-              expect(results).to match_array([cloud_network])
-              expect(results.first.ext_management_system).to eq(network_manager)
+        NETWORK_MODELS = [
+          CloudNetwork,
+          CloudSubnet,
+          FloatingIp,
+          LoadBalancer,
+          NetworkPort,
+          NetworkRouter,
+          SecurityGroup,
+          ManageIQ::Providers::NetworkManager
+        ].freeze
```

Comment:
Sorry @lpichler, I wasn't clear.  Can you do `described_class::NETWORK_MODELS_FOR_BELONGSTO_FILTER` +  `ManageIQ::Providers::NetworkManager` so we're not duplicating the hardcoded list here and the code?

Reply:
👍 oh yes sure, I read it badly